ecs/spotfleet-cluster/spotrequest-builder/launch_specification.template

```diff
@@ -0,0 +1,45 @@
+launch_specification {
+  ami                    = "${var.ami}"
+  instance_type          = "{{instance_type}}"
+  subnet_id              = "{{subnet_id}}"
+  vpc_security_group_ids = ["${aws_security_group.ecs-instances.id}"]
+  weighted_capacity      = {{weighted_capacity}}
+
+  tags = {
+    "Name" = "${var.prefix}-spot-instance"
+  }
+
+  iam_instance_profile = "${aws_iam_instance_profile.ecs.name}"
+
+  user_data = <<USER_DATA
+#!/bin/bash
```

Comment:
Please let `user_data` be a cloud-config like [this](https://github.com/TeliaSoneraNorge/telia-terraform-modules/blob/spotfleet/ecs/cluster/cloud-config.yml) :)